src/Umbraco.Infrastructure/Scoping/LegacyIScope.cs

```diff
@@ -1,12 +1,24 @@
 using System;
+using Umbraco.Cms.Core.Cache;
 using Umbraco.Cms.Core.Events;
+using Umbraco.Cms.Infrastructure.Persistence;
 
 // ReSharper disable once CheckNamespace
 namespace Umbraco.Cms.Core.Scoping;
 
 [Obsolete("Please use Umbraco.Cms.Infrastructure.Scoping.IScope or Umbraco.Cms.Core.Scoping.ICoreScope instead.")]
 public interface IScope : Infrastructure.Scoping.IScope
 {
+    /// <summary>
+    /// Gets the scope database.
+    /// </summary>
+    IUmbracoDatabase Database { get; }
```

Comment:
These additions are all intentionally hiding the inherited members, could you add new to prevent the compiler warnings for CS0108?

Reply:
Yep there's one property hiding inherited members -  `IScopeContext? Context` on IScopeProvider,

Reply:
Ok, added `new` keyword to `LegaxyIScope` and `IScopeProvider` 👍